docker/entrypoint.sh

```diff
@@ -57,3 +57,5 @@ harvest -c "$HARVEST_CFG_FILE"
 # Load data into Elasticsearch
 echo "Loading harvested data into Elasticsearch ..." 1>&2
 registry-manager load-data -dir /tmp/harvest/out/ -es "$ES_URL" -force
+registry-manager set-archive-status -status archived -lidvid urn:nasa:pds:insight_rad::2.1 -es "$ES_URL"
```

Comment:
If this newly added line is only applicable to test data, then we should have a check for,

` if [ "$RUN_TESTS" = "true" ]; then`